lib/state_of_the_nation.rb

```diff
@@ -30,6 +30,14 @@ def until(finish_key)
           (finish.blank? || round_if_should(finish) > round_if_should(time)) && round_if_should(start) <= round_if_should(time)
         end
 
+        define_method "active_in_interval?" do |interval_start:, interval_end: nil|
```

Comment:
I'm unsure about naming args here. The `active?` method doesn't take a named param and neither does `from` or `until` so I'm unsure if I should follow the library convention.

`considered_active` does have an `ignore_empty:` flag though, and since there are 2 dates to be provided, I like the explicitness of calling out the start and end.

Reply:
I think the single args to `from`, `until` and `active?` are pretty clear due to the method naming and I'd argue the same for `active_in_interval?` if we made `interval_end` required.